lib/Backend/LowerMDShared.cpp

```diff
@@ -5810,6 +5809,9 @@ LowererMD::GenerateCFGCheck(IR::Opnd * entryPointOpnd, IR::Instr * insertBeforeI
             IR::AddrOpnd * endAddressOfSegmentConstOpnd = IR::AddrOpnd::New(endAddressOfSegment, IR::AddrOpndKindDynamicMisc, m_func);
```

Comment:
Unrelated to this change, I noticed we add `InProcJITThunkEmitter::TotalThunkSize` to `jitThunkStartAddress`. It does look like it is "impossible" for this to overflow assuming the allocation was done properly. But since it is an assumption, we probably should `Assert(jitThunkStartAddress + InProcJITThunkEmitter::TotalThunkSize > jitThunkStartAddress)` or even do a runtime overflow check although I doubt it's necessary.